easybuild/easyconfigs/t/trimAl/trimAl-1.4.1-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'MakeCp'
+
+name = 'trimAl'
+version = '1.4.1'
+
+homepage = 'https://github.com/scapella/trimal'
+description = """EVB, FEP and LIE simulator."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+toolchainopts = {'pic': True}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['cb8110ca24433f85c33797b930fa10fe833fa677825103d6e7f81dd7551b9b4e']
+
+start_dir = 'source'
+
+files_to_copy = [(['trimal', 'readal', 'statal'], 'bin')]
+
+github_account = 'scapella'
```

Comment:
@omula can you move this up above `source_urls` (makes it clearer what it's for...)

Reply:
Done